Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -383,6 +387,22 @@ public void SpeakText(string text)
         }
 
         /// <summary>
+        /// Returns info whether a text is submitted and being processed by PlaySpeech method
+        /// Handy for avoiding situations when a text is submitted, but audio clip is not yet build (audiosource is not playing it yet!)
```

Comment:
Let's word it this way:
"Handy for avoiding situations when a text is submitted, but audio clip is not yet ready because the audio source isn't playing yet."